_pages/home.md

```diff
@@ -22,30 +22,65 @@ scientific Python packages. We also build technical capacity by providing a
-* Chris Holdgraf, Project Jupyter, Berkeley - BIDS.
-* Luiz Irber, DIB Lab -- UC Davis
-* Paige Bailey, Google
-* Kylen Solvik, Graduate Student -- Earth Lab, University of Colorado -- Boulder.
-* Carson Farmer, External advisor -- Textile.io.
-* Leonardo Uieda, Visiting Researcher -- University of Hawai'i at Mānoa -- Honolulu.
-* Filipe Fernandes, IOOS
+1. **[Join a community meeting](#community-meetings)**: We have monthly video meetings that you are welcome to attend.
+2. **[Join our community forum](https://pyopensci.discourse.group/)**. This is an online space
+   for discussion within the pyOpenSci community. Announcements for new meetings will be posted here.
+3. **Submit a package to pyOpenSci**. We are looking for Python packages that
+   support scientific Python community. Read more about our
+   [review process and scope](https://www.pyopensci.org/dev_guide/peer_review/aims_scope.html)
+4. **Volunteer to be a reviewer for pyOpenSci**. We are looking for people who
+   can help review software packages as they are submitted.
+5. **Help with technical infrastructure**. pyOpenSci has a few pieces of technical
+   infrastructure (like this site) which need maintenance and development.
+6. **Spread the word**! pyOpenSci is a community-driven project. Tell your friends,
+   recommend that a colleague submit a package, mention us at conferences, spread the word!
+
+### Community meetings
+
+Currently, the pyOpenSci team holds community meetings roughly every 2 weeks. These are
+held remotely, and open to anyone who would like to attend. We discuss ideas for improving
+the process / technology / community management / etc of the pyOpenSci community, and
+would value your participation!
+
+The best place to find information about the next community meeting is
+[in the community forum](https://pyopensci.discourse.group). We post dates / times
+for new meetings there.
+
+### Who's Involved
+
+See our [contributors page](contributors.html) for a list of the contributors
+in the pyOpenSci community.
+
+## FAQ
+
+### What's the difference between pyOpenSci and JOSS?
+
+The [Journal of Open Source Software](https://joss.theoj.org/) is a community dedicated
+to improving the visibility and quality of scientific software. They do
+so by providing a lightweight review and publishing process so that authors
+of packages can publish their packages with a DOI and citable artifact.
+They intentionally try to avoid lengthy review and feedback cycles
+with the goal of streamlining the publishing process.
+
+pyOpenSci encourages a more hands-on approach in the review process, and
+focuses more around software best-practices than publishing.
+We encourage our reviewers to provide more hands-on assistance and
+give more in-depth feedback than is typically required in JOSS. The goal
+of pyOpenSci is to ensure that all of its packages meet a minimum level
+of best practices and standards, and our review processes often take longer
```

Comment:
@choldgraf the one big difference is pyopensci accepts API wrappers. JOSS does not. so pyopensci accepts MORE packages than JOSS does .

Reply:
cool - feel free to either make edits directly, or I can make edits when I'm back at home (out of the office hanging out w/ family today)

Reply:
wonderful. ok i'll make a few edits probably after the meeting. if leo or luiz or lindsay are there, they can help review the language!!! if not i can bug @arfon who might also be willing to look at this PR and ensure we are describing JOSS in the way he'd like us to!!